### PR TITLE
V2IMP

### PR DESCRIPTION
**Pull request by pierallard**
This PR groups a loooot of little fixes to clean the V2 front.
See the commits labels to find this list.

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added Behats                      | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | Todo
| Migration script                  | -
| Tech Doc                          | -

### CODE REVIEW

File: features/import/product/import_products_and_download_imported_file.feature [hunk 1]
```diff
@@ -31,4 +31,5 @@ Feature: Execute a job
     When I am on the "csv_footwear_product_import" import job page
     And I launch the import job
     And I wait for the "csv_footwear_product_import" job to finish
-    Then I should see the secondary action "Download read files"
+    Then I should see the text ""
```
Comment (nidup) [inline features/import/product/import_products_and_download_imported_file.feature]: useless Then I should see the text "" ?

File: src/Oro/Bundle/FilterBundle/Form/Type/Filter/TextFilterType.php [hunk 2]
```diff
@@ -55,7 +55,6 @@ public function configureOptions(OptionsResolver $resolver)
             self::TYPE_NOT_CONTAINS => $this->translator->trans('oro.filter.form.label_type_not_contains'),
             self::TYPE_EQUAL        => $this->translator->trans('oro.filter.form.label_type_equals'),
             self::TYPE_STARTS_WITH  => $this->translator->trans('oro.filter.form.label_type_start_with'),
-            self::TYPE_ENDS_WITH    => $this->translator->trans('oro.filter.form.label_type_end_with'),
```
Comment (nidup) [inline src/Oro/Bundle/FilterBundle/Form/Type/Filter/TextFilterType.php]: :+1:

File: src/Pim/Bundle/EnrichBundle/Resources/config/form_extensions/product/index.yml [hunk 3]
```diff
@@ -8,6 +8,8 @@ extensions:
                 gridName: product-grid
         forwarded-events:
             grid_load:start: grid_load:start
+            grid_load:complete: grid_load:complete
+# TODO Do the same for assets/published products
```
Comment (nidup) [inline src/Pim/Bundle/EnrichBundle/Resources/config/form_extensions/product/index.yml]: TODO?

File: src/Pim/Bundle/UIBundle/Resources/public/images/logo_login.svg [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+		c1.2,16,0.9,32.1-1.1,44.8c-2.3,15.4-8.5,33.1-16.7,49.1c-17.9-2.1-36-6.8-50.1-13.4C26.1,75,12.7,66.5,0,56.5
+		c0.1,1.3,0.2,2.7,0.3,4c0,0.4,0.1,0.8,0.1,1.2c0.1,1,0.2,1.9,0.3,2.8c0,0.4,0.1,0.9,0.1,1.3c0.1,1,0.2,1.9,0.3,2.8
+		c0,0.4,0.1,0.8,0.1,1.2c0.2,1.2,0.3,2.4,0.5,3.6c0,0.1,0,0.2,0,0.4c0.2,1.4,0.4,2.8,0.6,4.1c0,0.1,0,0.2,0.1,0.3
+		c0.2,1.2,0.4,2.4,0.6,3.7c0,0.3,0.1,0.5,0.1,0.7c0.2,1.1,0.4,2.2,0.6,3.3c0.1,0.2,0.1,0.4,0.1,0.6c0.3,1.3,0.5,2.5,0.8,3.8l0,0
+		c0.9,4,2,7.9,3.1,11.6c0,0.1,0,0.1,0,0.1c1.9,6.2,4.2,12,6.7,17.1c0.4,0.7,0.8,1.5,1.1,2.2v0.1l0,0c0.3,0.6,0.7,1.3,1.1,1.9l0,0
+		c0,0,0,0,0,0.1c0.4,0.7,0.8,1.3,1.2,1.9l0,0c0.1,0.1,0.1,0.2,0.2,0.2l0,0c0.5,0.8,1.1,1.6,1.6,2.3c0.1,0.1,0.2,0.2,0.3,0.4l0.1,0.1
+		c0.6,0.8,1.3,1.6,1.9,2.4l0,0c0.5,0.5,0.9,1,1.4,1.5c0.2,0.2,0.4,0.4,0.6,0.6c0.3,0.3,0.6,0.6,0.9,0.9c0.2,0.2,0.4,0.4,0.7,0.6
+		c0.3,0.3,0.6,0.5,0.9,0.7c0.2,0.2,0.5,0.4,0.7,0.5c0.3,0.2,0.6,0.5,0.9,0.7c0.2,0.2,0.5,0.3,0.7,0.5c0.3,0.2,0.6,0.4,1,0.6
+		c0.2,0.1,0.5,0.3,0.7,0.4c0.4,0.2,0.7,0.4,1.1,0.5c0.2,0.1,0.4,0.2,0.6,0.3c0.6,0.3,1.2,0.5,1.7,0.7c0.4,0.1,0.7,0.3,1,0.4
+		c28.1,8.4,56.1-2.5,89.9,23.4"/>
```
Comment (nidup) [inline src/Pim/Bundle/UIBundle/Resources/public/images/logo_login.svg]: sure? :trollface:

File: src/Pim/Bundle/AnalyticsBundle/Resources/views/SystemInfo/index.html.twig [hunk 5]
```diff
@@ -25,4 +25,38 @@
             {% endfor %}
         </table>
     </div>
+
+    <script>
+        require(
```
Comment (SamirBoulil) [inline src/Pim/Bundle/AnalyticsBundle/Resources/views/SystemInfo/index.html.twig]: I imagine you couldn't create a proper JS file for this

File: src/Pim/Bundle/UIBundle/Resources/public/less/components/TitleContainer.less [hunk 6]
```diff
@@ -68,6 +68,9 @@
     justify-content: flex-end;
     flex-grow: 1;
     align-items: flex-start;
+    flex-wrap: wrap;
```
Comment (SamirBoulil) [inline src/Pim/Bundle/UIBundle/Resources/public/less/components/TitleContainer.less]: 🌮

File: src/Pim/Bundle/EnrichBundle/Resources/public/templates/job-execution/download-archives-buttons.html [hunk 7]
```diff
@@ -1,25 +1,19 @@
...
-                    <div>
-                        <a class="AknDropdown-menuLink"
-                           href="<%- generateRoute({id: executionId, archiver: archiver, key: key }) %>">
-                            <%- key %>
-                        </a>
-                    </div>
-                <% }); %>
-            </div>
+<% if (Object.keys(files).length === 1) { %>
+<a class="AknButton AknButton--action" title="<%- label %>" href="<%- Object.values(files)[0] %>">
```
Comment (SamirBoulil) [inline src/Pim/Bundle/EnrichBundle/Resources/public/templates/job-execution/download-archives-buttons.html]: if files is empty, doesn't it break ?

File: features/export/export_products_and_download_exported_file.feature [hunk 8]
```diff
@@ -46,4 +46,4 @@ Feature: Export and download exported products file
     tshirt-black;;men_2013,men_2014,men_2015;black;;;;usa;;;;;;"Ein elegantes schwarzes T-Shirt";"An elegant black t-shirt";"A stylish black t-shirt";"Un T-shirt noir élégant";1;tshirts;;0;;;;;;american_apparel;cotton;"Schwarzes T-Shirt";"Black t-shirt";"Black t-shirt";"T-shirt noir";;10.00;9.00;15.00;;size_L;files/tshirt-black/thumbnail/SNKRS-1C-t.png;;;;
     """
     Then I should see the text "Download generated files"
-    And I should see the text "Download generated archive"
+    Then I press "Download generated archive" on the "Download generated files" dropdown button
```
Comment (SamirBoulil) [inline features/export/export_products_and_download_exported_file.feature]: And

File: src/Pim/Bundle/AnalyticsBundle/Resources/views/SystemInfo/index.html.twig (hunk 5 above)
Comment (pierallard) [reply src/Pim/Bundle/AnalyticsBundle/Resources/views/SystemInfo/index.html.twig]: Nope. In fact, it's the only way to call JS FormExtensions into twig files for now...

File: src/Pim/Bundle/UIBundle/Resources/public/images/logo_login.svg (hunk 4 above)
Comment (pierallard) [reply src/Pim/Bundle/UIBundle/Resources/public/images/logo_login.svg]: You're right, it's .5,0.9,0.7c0.2,0.2,0.5,0.4,0.7,0.5c0.3

**Pull request merged**